spec/lib/govuk_component/inset_text_helpers_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+RSpec.describe GovukComponent::InsetTextHelpers, type: :helper do
+  include RSpecHtmlMatchers
+
+  describe '#govuk_inset_text' do
+    subject(:markup) { helper.govuk_inset_text() { 'My content' } }
+
+    it 'adds inset text with govuk class' do
+      is_expected.to have_tag(:div, with: { class: 'govuk-inset-text' })
+    end
+
+    it 'yields content to div tag' do
+      is_expected.to have_tag(:div, with: { class: 'govuk-inset-text' }, text: 'My content')
+    end
+
+    context 'with custom classes' do
+      subject(:markup) do
+        helper.govuk_inset_text(class: 'my-custom-class1 my-custom-class2') do
```

Comment:
**not a blocker** but for short blocks i would not use `do end`, unless rubocop is complaining about line length (sigh)

```
helper.govuk_inset_text(class: 'my-custom-class1 my-custom-class2') { 'My content' }
```